src/util.ts

```diff
@@ -1,87 +1,93 @@
-/*-----------------------------------------------------------------------------------------------
+/* -----------------------------------------------------------------------------------------------
  *  Copyright (c) Red Hat, Inc. All rights reserved.
  *  Licensed under the MIT License. See LICENSE file in the project root for license information.
  *-----------------------------------------------------------------------------------------------*/
 
-type RunnerOS = 'Linux' | 'macOS' | 'Windows';
+type RunnerOS = "Linux" | "macOS" | "Windows";
 
 namespace Utils {
-  export const INPUT_OC_VERSION = 'oc_version';
+    export const INPUT_OC_VERSION = "oc_version";
 
-  export const LATEST = 'latest';
+    export const LATEST = "latest";
 
-  let runnerOS: RunnerOS | undefined;
-  export function getRunnerOS(): RunnerOS {
-    if (runnerOS) {
-      return runnerOS;
-    }
+    let runnerOS: RunnerOS | undefined;
+    export function getRunnerOS(): RunnerOS | undefined {
```

Comment:
where is the case that this returns `undefined`?

Reply:
I made return type `undefined` after seeing this code 
```
runnerOS = process.env.RUNNER_OS as RunnerOS | undefined;
```
I couldn't think of any real use case, just added `undefined` to avoid any breaking change in case I miss some usecase